package.json

```diff
@@ -32,7 +32,7 @@
     "Edge"
   ],
   "engines": {
-    "node": "^20.17.0 <21.0.0",
+    "node": ">=20.17.0 <22.15.0",
```

Comment:
I widened the acceptable version of `node` to be more in sync with Spacecat.  It isn't as easy to switch versions on Windows, and I get tired of uninstalling/reinstalling node, and get extra nervous in my on-call week.